vertx-web/src/main/java/io/vertx/ext/web/handler/sockjs/impl/EventBusBridgeImpl.java

```diff
@@ -342,7 +342,8 @@ private void handleSocketException(SockJSSocket sock, Throwable err, Map<String,
     if (err != null) {
       msg.put("message", err.getMessage());
     }
-    checkCallHook(() -> new BridgeEventImpl(BridgeEventType.SOCKET_CLOSED, msg, sock));
+    // Maybe a new SOCKET_ERROR ????
+    // checkCallHook(() -> new BridgeEventImpl(BridgeEventType.SOCKET_CLOSED, msg, sock));
```

Comment:
We had a discussion in 4.1.1:

https://github.com/vert-x3/vertx-web/issues/1986

Maybe the reason that having 2 terminal states is valid, and we should get that back?

Reply:
By commenting this, then, there's no way to signal that the state is exception/closed

Reply:
closed will happen whatsoever with the new close handler that is guaranteed to be called.

with a socket_error event application could be aware of error